modules/mod_ginger_base/controllers/controller_search.erl

```diff
@@ -2,10 +2,10 @@
 -module(controller_search).
 
 -export([
-    init/1,
-    content_types_provided/2,
-    to_json/2
-]).
+	 init/1,
+	 content_types_provided/2,
+	 to_json/2
+	]).
```

Comment:
Please don’t make unnecessary whitespace changes. The old code matches [Inaka’s guidelines](https://github.com/inaka/erlang_guidelines), which we follow.